README.md

```diff
@@ -73,15 +79,15 @@ VMClarity uses a pluggable scanning infrastructure to provide:
 - Rootkit detection
 
 The pluggable scanning infrastructure uses several tools that can be
-enabled/disabled on an individual basis. VMClarity normalizes, merges and
+enabled/disabled on an individual basis. OpenClarity normalizes, merges and
 provides a robust visualization of the results from these various tools.
 
 These tools include:
 
 - SBOM Generation and Analysis
   - [Syft](https://github.com/anchore/syft)
   - [Trivy](https://github.com/aquasecurity/trivy)
-  - [Windows Registry](cli/analyzer/windows)*
```

Comment:
btw widnows registry scanner is supporetd for all hosts (windows, unix, mac)

Reply:
Should I add something here?

Reply:
removing the `*` and `* Windows only` from the readme should be enough